Makefile

```
@@ -15,7 +15,7 @@ cs-dry-run:
 	vendor/bin/php-cs-fixer fix --config=.php_cs --verbose --diff --dry-run
 
 cs-fix:
-	vendor/bin/php-cs-fixer fix --config=.php_cs
+	vendor/bin/php-cs-fixer fix --config=.php_cs.dist
```

We could omit this entirely:

```suggestion
	vendor/bin/php-cs-fixer fix
```

and let the auto-select do its magic